pkg/clusters/utils.go

```diff
@@ -75,3 +75,31 @@ func GetIngressLoadbalancerStatus(ctx context.Context, c Cluster, namespace stri
 		return nil, fmt.Errorf("%T is not a supported ingress type", ingress)
 	}
 }
+
+// CreateNamespace create customized namespace
+func CreateNamespace(ctx context.Context, cluster Cluster, namespace string) error {
+	nsName := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+
+	fmt.Printf("creating namespace %s.", namespace)
```

Comment:
Similar to above, we should avoid printing to stdout in a public package that's meant to be used as a lib.
```suggestion
```